site/src/pages/TaskPage/TaskPage.tsx

```diff
@@ -275,9 +276,22 @@ const WorkspaceNotRunning: FC<WorkspaceNotRunningProps> = ({
 					<span className="text-content-secondary text-sm">
 						Apps and previous statuses are not available
 					</span>
+					{workspace.outdated && (
+						<div
+							data-testid="workspace-outdated-tooltip"
+							className="flex items-center gap-1.5 mt-1 text-content-secondary text-sm"
+						>
+							<WorkspaceOutdatedTooltip workspace={workspace}>
+								<span>
+									You can update your task workspace to a newer version
+								</span>
```

Comment:
The component already wraps the children in a span

```suggestion
								You can update your task workspace to a newer version
```